src/sid/contacts.py

```diff
@@ -217,7 +221,12 @@ def calculate_infections_by_contacts(
     )
     newly_infected = pd.Series(combined_newly_infected, index=states.index)
 
-    return newly_infected, n_has_additionally_infected, missed_contacts, was_infected_by
+    return (
+        newly_infected,
+        n_has_additionally_infected,
+        missed_contacts,
+        was_infected_by,
+    )
```

Comment:
Revert. The comma after ``was_infected_by`` prevents black from formatting a single line.